lib/cylc/scheduler.py

```diff
@@ -1018,10 +806,14 @@ def command_stop_now(self):
         raise SchedulerStop("Stopping NOW")
 
     def command_set_stop_after_point(self, point_string):
+        """Set stop after ... point."""
         self.set_stop_point(self.get_standardised_point_string(point_string))
 
     def command_set_stop_after_clock_time(self, arg):
-        # format: ISO 8601 compatible or YYYY/MM/DD-HH:mm (backwards comp.)
+        """Set stop after clock time.
+
+        format: ISO 8601 compatible or YYYY/MM/DD-HH:mm (backwards comp.)
```

Comment:
Side note: no need for backwards compatibility any more (`cylc gui` won't output in this form, and someone would have to deliberately ignore the command line help for cylc stop on the command line).